build.gradle

```diff
@@ -1,29 +1,65 @@
 import groovy.json.JsonSlurper
 import groovy.transform.Canonical
 
+plugins {
+    id "io.spring.dependency-management" version "1.0.3.RELEASE" apply false
+
+    id 'com.diffplug.gradle.spotless' version '2.4.1' apply false
+    id 'net.ltgt.apt' version '0.13' apply false
+    id 'com.commercehub.cucumber-jvm' version '0.12' apply false
+}
+
 group 'com.amazonaws.blox'
 version '0.1-SNAPSHOT'
 description "Blox: Open Source schedulers for Amazon ECS"
 
-buildscript {
+allprojects {
+    apply plugin: 'com.diffplug.gradle.spotless'
+
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        mavenCentral()
     }
+}
+
+subprojects {
+    apply plugin: 'io.spring.dependency-management'
 
-    dependencies {
-        classpath 'com.diffplug.gradle.spotless:spotless:2.4.1'
+    dependencyManagement {
+        imports {
+            mavenBom 'com.amazonaws:aws-java-sdk-bom:1.11.215'
```

Comment:
do we need both sdk version or is this so we don't have to refactor code that doesn't use v2 now?

Reply:
Yes, this is so that we don't have to refactor the parts of the dataservice that still depend on V1 now.